### DISCUSSION TITLE
v0.25.8 conda

### DISCUSSION THREAD

**Discussion by griffiri** [CamDavidsonPilon/lifelines#1208] [General]
hi @CamDavidsonPilon - could you make 0.25.8 available on conda forge? Would like to test serialization feature.

thanks!

**Comment by CamDavidsonPilon** [comment]
Hey, I'm blocked formulaic not being on conda-forge: https://github.com/conda-forge/lifelines-feedstock/pull/92/

**Reply by griffiri** [reply]
Ah ic - is there a plan to get it on there?

**Reply by CamDavidsonPilon** [reply]
0.25.9 Should be available now!
